src/tests/ftest/util/file_utils.py

```diff
@@ -68,6 +68,31 @@ def change_file_owner(logger, hosts, filename, owner, group, timeout=15, verbose
     return __run_command(logger, hosts, command, verbose, timeout)
 
 
+def create_owned_directory(logger, hosts, directory, timeout=15, verbose=True, owner=None,
```

Comment:
Couldn't we add an argument to create_directory to handle this? This is the same function but conditionally calls chown afterwards

Reply:
Yes, that's a good idea.